docs/RELEASE_NOTES.md

```diff
@@ -27,6 +27,15 @@
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
 
+### 5.1.1 - May 7, 2021
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/5.1.0...5.1.1) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/5.1.1/) |
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/5.1.1)
+
+- 🐞 **Bug Fixes** (1)
+    - **Product Sync** - Fixed a bug in the `ProductSync` related to querying of productProjections based on key values, when key value contains special characters(") [#269](https://github.com/commercetools/commercetools-project-sync/issues/269)
+    - Same fix also done for GraphqlQuery search based on keys.
```

Comment:
This is not a separate issue we fixed in this release. So, it would be better to include this point to the previous sentence or remove this point as it is obvious.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/727/commits/f9dc9631f8a1221bad15ea5f2b4586c2ce621df2 This is also related to ProductDraft key value, So removed it.